pandas/tools/tile.py

```diff
@@ -249,16 +259,24 @@ def _format_levels(bins, prec, right=True,
             if a != b and fa == fb:
                 raise ValueError('precision too low')
 
-            formatted = '(%s, %s]' % (fa, fb)
+            if time_data:
```

Comment:
@jreback i have moved this handling down to the formatter in the latest iteration of this code